releases/R2024b/README.md

```diff
@@ -0,0 +1,286 @@
+# MATLAB Web App Server on Amazon Web Services - R2024b
+
+# Deployment Steps
+Follow these steps to deploy the R2024b MATLAB Web App Server reference architecture on AWS. To deploy reference architectures for other releases, see [Deploy Reference Architecture for Your Release](/README.md#deploy-reference-architecture-for-your-release). 
+## Step 1. Launch Template
+Before launching the template, make sure that you have selected one of these supported AWS regions from the top navigation:<ul><li>**US East (N. Virginia)**</li><li>**Europe (Ireland)**</li><li>**Asia Pacific (Tokyo)**</li></ul>
+
+Then, click the appropriate **Launch Stack** button to launch the stack configuration template for deploying resources onto your AWS account. You can deploy resources onto either a new or existing VPC.
+
+| Release | New VPC | Existing VPC | Operating Systems |
+|---------|---------| ------------ | ----------------- |
+| R2024b | <a href="https://us-east-2.console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/quickcreate?templateURL=https://mb-webapps-template.s3.us-east-2.amazonaws.com/WebAppServer_new.yml" target="_blank">     <img src="https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png"/> </a> | <a  href ="https://us-east-2.console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/create/review?templateURL=https://mb-webapps-template.s3.us-east-2.amazonaws.com/WebAppServer_existing.yml"  target ="_blank" >      <img  src ="https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png" /> </a> | Windows Server 2022 or Ubuntu 22.04 VM |
```

Comment:
Please update the urls to new VPC and existing VPC template URLs, respectively that are mentioned in templates.json.